src/locales/zh.json

```diff
@@ -0,0 +1,92 @@
+{
```

Comment:
Some keys are now missing, can you recreate the `zh.json` file based on the current `en.json` file from master?

Reply:
Forget about this the keys has been updated with the chinese translations